app/domain/operations/individual/calculate_monthly_aggregate.rb

```diff
@@ -36,6 +36,7 @@ def consumed_aptc_amount(values)
         @subscriber_applicant_id = values[:subscriber_applicant_id]
 
         aptc_enrollments = HbxEnrollment.yearly_aggregate(@family.id, @effective_on.year)
+        aptc_enrollments = aptc_enrollments.reject{|enr| enr.effective_on > @effective_on && enr.subscriber.applicant_id == @subscriber_applicant_id}
```

Comment:
Enrollments that have the same subscriber, future effective on compared to new effective on are excluded for consumed APTC calculation as these enrollments will be canceled.